main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"database/sql"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	_ "modernc.org/sqlite"
+
+	"github.com/go-chi/chi"
+)
+
+const defaultPort = "7540"
+const webDir = "./web"
+const fileDb = "scheduler.db"
+
+func main() {
+	// запрашиваем подключение к БД - проверяем наличия файла
+	appPath, err := os.Executable()
+	if err != nil {
+		log.Fatal(err)
+	}
+	// Формируем путь к файлу базы данных
+	dbFile := filepath.Join(filepath.Dir(appPath), fileDb)
+	// Проверяем, существует ли файл базы данных
+	_, err = os.Stat(dbFile)
+	var install bool
+	if err != nil {
+		install = true
+		log.Println("требуется создания файла БД ", fileDb, install)
+	}
+	db, err := sql.Open("sqlite", dbFile)
+	if err != nil {
+		log.Fatal("Ошибка при открытии базы данных:")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		log.Fatal("Ошибка при открытии базы данных:", err)
```